tests/Makefile

```diff
@@ -21,11 +30,13 @@ LOCAL_CFLAGS+=$(THIRDPARTY_INCLUDES)
 LOCAL_CFLAGS += -D_POSIX_C_SOURCE=200809L
 
 ifndef SHARED
-DEP=../library/libmbedcrypto.a ../library/libmbedx509.a ../library/libmbedtls.a
+MBEDLIBS=../library/libmbedcrypto.a ../library/libmbedx509.a ../library/libmbedtls.a
 else
-DEP=../library/libmbedcrypto.$(DLEXT) ../library/libmbedx509.$(DLEXT) ../library/libmbedtls.$(DLEXT)
+MBEDLIBS=../library/libmbedcrypto.$(DLEXT) ../library/libmbedx509.$(DLEXT) ../library/libmbedtls.$(DLEXT)
 endif
 
+MBEDTESTSLIB=./src/libmbedtests.a
```

Comment:
It would be good to add a comment stating that this is deliberately linked statically even under `SHARED`. Likewise in `CMakeLists.txt`.

Reply:
This code has been removed.